### PR TITLE
docs: update README.md

### PR DESCRIPTION
**Pull request by rchincha**

### CODE REVIEW

**Comment by rchincha** [comment]
cc: @SteveLasker @afflom @sabre1041 @ktarplee 

Cleaned up the home page so folks get a better idea of what we are trying to do.

File: README.md [hunk 1]
```diff
@@ -31,15 +31,33 @@ Old working document of the [charter](https://docs.google.com/document/d/1w_lo2R
...
 
-The CNCF Artifacts Working Group is under the App Delivery Technical Advisory Group hosted by the [Cloud Native Computing Foundation (CNCF)](https://cncf.io).
-
-`https://github.com/cncf/tag-app-delivery/tree/main/artifacts-wg`
+The CNCF Artifacts Working Group is under the [App Delivery Technical Advisory Group](https://github.com/cncf/tag-app-delivery/tree/main/artifacts-wg) hosted
+by the [Cloud Native Computing Foundation (CNCF)](https://cncf.io).
 
 ## Chairs
 
 - Alex Flom (@afflom)
```
Comment (sabre1041) [inline README.md:39]: Should this be updated based on https://github.com/cncf/tag-app-delivery/issues/516?

File: README.md [hunk 2]
```diff
@@ -51,6 +69,16 @@ include:
 * basic prerequisites
 -->
 
+Currently, the working group is evaluating what the current state of various
```
Comment (sabre1041) [inline README.md]:
```suggestion
Currently, the working group is evaluating the current state of various
```

File: README.md [hunk 3]
```diff
@@ -51,6 +69,16 @@ include:
...
+software ecosystems is and whether it is feasible to standardize and also
+suggest changes.
+
+git clone `https://github.com/cncf-tags/wg-artifacts.git`
```
Comment (sabre1041) [inline README.md]: There should be an overview of what these commands are attempting to achieve

File: README.md [hunk 4]
```diff
@@ -64,27 +92,7 @@ started contributing, please see our [Contributor Guide](CONTRIBUTING.md).
...
-tools. The following specific functionality will therefore not be incorporated:
-
-- [TODO: Excluded function 1]
-- [TODO: Excluded function 2]
-
-[TODO: PROJECTNAME] implements [TODO: List of major features, existing or
-planned], through [TODO: Implementation
-requirements/language/architecture/etc.]. It will not cover [TODO: short list
-of excluded items]
+- Define a set of query interfaces that can standardized across various software ecosystems.
```
Comment (sabre1041) [inline README.md:100]: Missing `### In Scope` section header

**Review by sabre1041 — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (rchincha) [reply README.md:100]: I removed that for now. Once things are more clear, we can add that again (at least that is my plan)
Less clutter!

File: README.md (hunk 1 above)
Comment (rchincha) [reply README.md:39]: Made this a separate PR
https://github.com/cncf-tags/wg-artifacts/pull/32

File: README.md (hunk 4 above)
Comment (rchincha) [reply README.md:100]: Let me know if you or others disagree.

**Review by SteveLasker — APPROVED** [review]
LGTM

**Comment by rchincha** [comment]
If there are no further objections, will merge this and we can iterate.

**Pull request merged**